README.md

```diff
@@ -101,3 +101,7 @@ Meilisearch is a search engine created by [Meili](https://www.welcometothejungle
 - For everything else, please check [this page listing some of the other places where you can find us](https://docs.meilisearch.com/learn/what_is_meilisearch/contact.html)
 
 Thank you for your support!
+
+## 📦 Internal crates and their versioning
```

Comment:
Hum, I don't really like this title, it's too technical compared to what we have on it. We are a famous project and now the README is almost like a landing page. However I don't want to block the PR, I will make a PR suggestion later
(I notice we don't talk about contribution in the README, it will be also an opportunity to add this 😄)